crates/build-utils/src/content_reader.rs

```diff
@@ -71,4 +75,31 @@ impl ContentReader {
         // Return a clone of the block
         Ok(first_block.clone())
     }
+
+    pub fn transaction(test_case: &Value, block: &Value) -> Result<Value, eyre::Error> {
```

Comment:
goal: get the tx sender later on and use it to get it's secret key from a known mapping